### DISCUSSION TITLE
kong-plugin-acme RSA key size setting

### DISCUSSION THREAD

**Discussion by lodrantl** [Kong/kong#8111] [Ideas and feature requests]
resty-acme supports setting the key size for the generated RSA certificate. Would it be possible to add such a configuration to the
kong-plugin-acme too?

Due to the limitations of some embedded devices we require RSA certificates with size 2048 and the hardcoded 4096 is unfortunately not possible. I can provide a PR if such a change would be accepted, we already have a patched version in production.

**Comment by fffonion** [comment]
@lodrantl That's a good feature, PRs are definitely welcomed!

**Comment by fffonion** [comment]
Link to https://github.com/Kong/kong/pull/8114

**Comment by onovaes** [comment]
I'm using version 2.0 and acme plugin

Is there any way to generate these certificates with 2048 bits?  (temporarily)
